### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by rachelcarmena**
Last fix!! Fix arrow-kt/arrow-optics#24 and arrow-kt/arrow-incubator#53 for Arrow Fx.

Note: I'm preparing something to avoid this kind of situations and to be able to check all the libraries in local workspace with an easy way :wink:

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -13,3 +13,7 @@ kotlin.incremental=true
 #Parallelism needs to be set to 1 since the concurrent tests in arrow-effects become flaky otherwise
 kotlintest.parallelism=1
 kapt.incremental.apt=false
+GENERIC_CONF=file:///home/rachel/lab/arrow-workspace/arrow/generic-conf.gradle
+SUBPROJECT_CONF=file:///home/rachel/lab/arrow-workspace/arrow/subproject-conf.gradle
+DOC_CONF=file:///home/rachel/lab/arrow-workspace/arrow/doc-conf.gradle
+PUBLISH_CONF=file:///home/rachel/lab/arrow-workspace/arrow/publish-conf.gradle
```
Comment (aballano) [inline gradle.properties]: leftover?

Comment (rachelcarmena) [reply gradle.properties]: A mistake!! Thanks for the review!!

Comment (rachelcarmena) [reply gradle.properties]: Removed! :sweat_smile:

**Comment by rachelcarmena** [comment]
Thanks @aballano :tada:

**Pull request merged**